versioned_docs/version-8.0/guides/update-guide/connectors/060-to-070.md

```diff
@@ -0,0 +1,17 @@
+---
+id: 060-to-070
+title: Update 0.6 to 0.7
+description: "Review which adjustments must be made to migrate from Connector SDK 0.6.x to 0.7.0."
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+With the Connector SDK [0.7.0 release](https://github.com/camunda/connector-sdk/releases/tag/0.7.0), we made
```

Comment:
Does this breaking change need to be noted anywhere else in the documentation for awareness?

Reply:
Yes, I clarified here and similar files.